Makefile

```diff
@@ -21,23 +21,27 @@ all: pkg tar
 pkg: deb rpm
 
 deb: populate-deb
+	(cd pkg/deb/; \
+		DEBMAIL="stackdriver-agents@google.com" DEBFULLNAME="Stackdriver Agents" \
+		dch --package "${PACKAGE_NAME}" \
+		-v "${PACKAGE_VERSION}" "Automated Build")
```

Comment:
Ah, now I see why we thought this was a good idea to do in our build scripts instead. Is `"Automated build"` always a good log message? Should we provide a mechanism to override it?

Reply:
Done. I chose `CHANGE_TEXT`, but there's probably a better name.